Fix batch update failures in the MCP files tool (#27121)
directus/directus: .changeset/mcp-files-update-data-schema.md

---
"@directus/api": patch
---

> Fixed MCP files tool schemas so update `data` accepts both a single object (with `keys`) and an array for batch updates

Reviewer: ```suggestion
Fixed batch update failures in the MCP files tool
```
